stylegan_code_finder/lightning_modules/base_lightning.py

```diff
@@ -0,0 +1,61 @@
+import os
+import torch
+from torch import nn
+import pytorch_lightning as pl
+from training_builder.base_train_builder import BaseTrainBuilder
+from networks.trans_u_net.utils import DiceLoss
+from torch.optim.lr_scheduler import CosineAnnealingWarmRestarts
+from utils.clamped_cosine import ClampedCosineAnnealingLR
+from visualization.segmentation_plotter_lightning import SegmentationPlotter
+
+
+class BaseSegmenter(pl.LightningModule):
+    def __init__(self, training_builder: BaseTrainBuilder, configs, segmentation_plotter: SegmentationPlotter, wandb_logger):

```

Comment:
`wandb_logger` does not have to be given as param. It is already stored in `self.logger` (lightning handles it)